mpas_analysis/shared/plot/plotting.py

```diff
@@ -141,6 +143,106 @@ def timeseries_analysis_plot(config, dsvalues, N, title, xlabel, ylabel,
         plt.close()
 
 
+
+def timeseries_analysis_plot_polar(config, dsvalues, N, title,
+                                   fileout, lineStyles, lineWidths, calendar,
+                                   titleFontSize=None, figsize=(15, 6), dpi=300):
+
+    """
+    Plots the list of time series data sets on a polar plot and stores
+    the result in an image file.
+
+    Parameters
+    ----------
+    config : instance of ConfigParser
+        the configuration, containing a [plot] section with options that
+        control plotting
+
+    dsvalues : list of xarray DataSets
+        the data set(s) to be plotted
+
+    N : int
+        the numer of time points over which to perform a moving average
+
+    title : str
+        the title of the plot
+
+    fileout : str
+        the file name to be written
+
+    lineStyles, lineWidths : list of str
+        control line style/width
+
+    titleFontSize : int, optional
+        the size of the title font
+
+    figsize : tuple of float, optional
+        the size of the figure in inches
+
+    dpi : int, optional
+        the number of dots per inch of the figure
+
+    Authors
+    -------
+    Adrian K. Turner
+
+    Last Modified
+    -------------
+    03/15/2017
+    """
+    plt.figure(figsize=figsize, dpi=dpi)
+
+    minDays = []
+    maxDays = []
+    for dsIndex in range(len(dsvalues)):
+        dsvalue = dsvalues[dsIndex]
+        if dsvalue is None:
+            continue
+        mean = pd.Series.rolling(dsvalue.to_pandas(), N, center=True).mean()
+        mean = xr.DataArray.from_series(mean)
+        minDays.append(mean.Time.min())
+        maxDays.append(mean.Time.max())
+        plt.polar((mean['Time']/365.0)*math.pi*2.0, mean,
+                 lineStyles[dsIndex],
+                 linewidth=lineWidths[dsIndex])
+
+    ax = plt.gca()
+
+    # set azimuthal axis formatting
+    majorTickLocs = np.zeros(12)
+    minorTickLocs = np.zeros(12)
+    majorTickLocs[0] = 0.0
+    minorTickLocs[0]= (constants.daysInMonth[0] * math.pi) / 365.0
+    for month in range(1, 12):
+        majorTickLocs[month] = majorTickLocs[month-1] + \
+            ((constants.daysInMonth[month-1] * math.pi * 2.0) / 365.0)
```

Comment:
@akturner, for some reason, people suggest using `np.pi` instead of `math.pi`.  I prefer it just because it tends to mean one less library to import but there is a more technical reason it's supposed to be better.  I could look it up but I'm too lazy...